packages/ember/index.js

```diff
@@ -10,8 +10,7 @@ import { FEATURES, isEnabled } from '@ember/canary-features';
 import * as EmberDebug from '@ember/debug';
```

Comment:
This in particular shows the improvement. 👏🏼 Also I hope to destroy this entire package sometime *soon-ish*. 😂